app/src/main/java/com/concordium/wallet/App.kt

```diff
@@ -1,30 +1,40 @@
 package com.concordium.wallet
 
 import android.content.Context
+import com.concordium.wallet.onboarding.ui.di.AppModule
+import com.concordium.wallet.onboarding.ui.di.OnboardingModule
 import com.concordium.wallet.util.Log
+import com.walletconnect.android.BuildConfig
 import com.walletconnect.android.Core
 import com.walletconnect.android.CoreClient
 import com.walletconnect.android.relay.ConnectionType
 import com.walletconnect.sign.client.Sign
 import com.walletconnect.sign.client.SignClient
+import org.koin.android.ext.koin.androidContext
+import org.koin.android.ext.koin.androidLogger
+import org.koin.core.context.GlobalContext.startKoin
 import org.matomo.sdk.TrackerBuilder
 import org.matomo.sdk.extra.MatomoApplication
 
-class App : MatomoApplication(){
+class App : MatomoApplication() {
 
     companion object {
+        //todo[SK] remove this, leaking context, use DI
```

Comment:
Again it's OK to add TODOs as long as you plan to address them "soon".

Reply:
yes, for now i have removed all mine todos and added to personal notes, will create tasks for these changes.